package.json

```diff
@@ -9,7 +9,7 @@
     "test": "nyc --statements 100 -- mocha -- --check-leaks --timeout 3000"
   },
   "dependencies": {
-    "aws-sdk": "~2.631.0"
+    "aws-sdk": "~2.1055.0"
```

Comment:
Why not?

```suggestion
    "aws-sdk": "^2.1055.0"
```

Reply:
The idea was to pin it to the version used in the AWS Lambda runtime environment per: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

I'll update it to match the current Node 16 runtime.